### PR TITLE
Travis cache

### PR DESCRIPTION
**Pull request by 3opk23**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ How to [secure your setup](/doc/security/ssl.md).
 Development guides are coming soon.
 
 ## License
-
+<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.
```
Comment (NGTmeaty) [inline README.md]: Why are we changing the license?

File: .travis.yml [hunk 2]
```diff
@@ -35,3 +35,4 @@ deploy:
   on:
     repo: codercom/code-server
     branch: master
+cache: yarn
```
Comment (NGTmeaty) [inline .travis.yml]: These change were already pushed? I fail to see how this PR is related to a "Travis cache"

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
+{
```
Comment (NGTmeaty) [inline .vscode/launch.json]: If were adding a .vscode, I think we'll want to make these slightly more unique to our project, perhaps in another commit?

File: .history/scripts/install-packages_20190317181527.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
+import { exec, execSync } from "child_process";
```
Comment (NGTmeaty) [inline .history/scripts/install-packages_20190317181527.ts]: This shouldn't be pushed.

File: .history/scripts/install-packages_20190317190816.ts [hunk 5]
```diff
@@ -0,0 +1,54 @@
+import { exec, execSync } from "child_process";
```
Comment (NGTmeaty) [inline .history/scripts/install-packages_20190317190816.ts]: This shouldn't be pushed.

**Comment by kylecarbs** [comment]
Going to close. Tackling the lengthy CI problem in the `ci` branch.

**Pull request closed without merging**